script.js

```diff
@@ -1,4 +1,65 @@
+//input = Player1, roll 2 dice, order of guess (dice1 or dice2)
+//output = Player2, output 2 guesses ... username2, output 2 guesses + outcome (who wins/lose)
+var randomWord = "Keyboard match";
+var gameMode = "enter username";
+var username = "";
+var myOutputValue = "";
+var Player1 = 0;
+var Player2 = 0;
+var numberOfTries = 0;
+
+var rollDice = function () {
+  var randomNumber = Math.floor(Math.random() * 7);
+  return randomNumber;
+};
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "hello world";
+  var myOutputValue = "Try Again";
+  if (gameMode == "enter username") {
+    console.log("username entered");
+    username = input;
+    gameMode = "game start";
+    myOutputValue = `Welcome ${username} <br><br> Press Submit to roll Dice. `;
+    console.log("dice game starts");
+    return myOutputValue;
+  }
+  if (gameMode == "game start") {
+    gameMode = "choose dice";
+    diceOne = rollDice();
+    diceTwo = rollDice();
+    dicePlayerOne = "" + diceOne + diceTwo;
+    myOutputValue = `Dice 1 rolled ${diceOne} and Dice 2 rolled ${diceTwo}. <br><br> Please choose order of dice guess by inputting 'Dice 1' or 'Dice 2' as the first dice.`;
+    console.log("2 dice rolled" + diceOne + diceTwo + dicePlayerOne);
+    return myOutputValue;
+  }
+
```

Comment:
Try to incorporate more comments to explain the design of your code, and what each line is trying to achieve. This makes re-reading your code much easier, esp after you've not read it for a long time.